aviary/wrenformer/run.py

```diff
@@ -0,0 +1,334 @@
+from __future__ import annotations
```

Comment:
This feels like it should be in examples rather than in the package? This appears to just be because SWA is hard to do in the base class?

Reply:
You can use it without an account. See the [Allegro tutorial notebook](https://colab.research.google.com/drive/1a0s34KERvJnvkdKSHkyG8HZYijC6xhwv?authuser=1) for an example of that. Needs a Google account for Colab but not a WandB account.

The bigger issue is that the base class would become even larger and harder to read.

Reply:
> I assume it's now used by both `mp_wbm` and `matbench`?

Correct. I'll move it to examples.